doc/api/esm.md

```diff
@@ -1301,7 +1326,7 @@ The resolver can throw the following errors:
 > 8. Otherwise,
 >    1. Set _format_ the module format of the content type associated with the
 >       URL _resolved_.
-> 9. Load _resolved_ as module format, _format_.
+> 9. Return _format_ and _resolved_
```

Comment:
I would still prefer to rename this to `_defaultFormat_` but don't have a strong opinion if you'd rather leave it as-is.

Reply:
We could also call it `formatHint`.

Reply:
Thanks. Notice the more interesting change here: I changed the "Load ..." to "Return...", because, IMO, the resolver algorithm does not do the loading, but rather the code above it.

(this goes back to our "resolver vs. default resolver algorithm" discussion.